Cookbook/Technical-Documents/ToolsAndServices.md

```diff
@@ -25,3 +16,31 @@
 | VPN | Remote Access to our intranet and internal domains | [IT Support ticket](https://supportbybabylon.atlassian.net/servicedesk/customer/portal/5/group/12/create/43) |
 | [CircleCI](https://circleci.com/gh/Babylonpartners) | See CI jobs and logs | Use your GitHub login |
 | [HockeyApp](https://rink.hockeyapp.net/) | see OTA builds sent internally to QA | Ask support engineer in the iOS team channel on Slack to invite you to the org |
+
+## Internal tools
+
+* Xcode templates
+
+  These templates are helpful when implementing new features or adding tests for existing ones. To install them run `Templates/install_xcode_templates.sh`
+  
+* Git hooks
+
+  If you want you can install additional git hooks which will check for pods and Xcode templates to be in sync when switching branches or pulling from remote. To install them follow instructions in `githooks/README.md`
+  
+* Slack bots
+  
+  You can trigger CI jobs from Slack in the `ios-build` channel using slash commands. Check [this guide](TBD) for details.
+  
+
+## Optional tools
+
+* [Charles](https://www.charlesproxy.com) is a tool that allows to intercept network trafic from the app
+
+  * Install Charles Proxy
+  * Add the desired URL under Proxy > SSL Proxy Settings
+  * Install on Simulator via Help > SSL Proxying > Install on iOS Simulator
+  * The license for Charles can be found in the iOS team vault on 1Password
+
+* [Insomnia](https://insomnia.rest) is a REST client app that is helpful when working with APIs
+
+* [Fork](https://forkapp.io) and [GitUp](https://gitup.co) are the git client apps we love
```

Comment:
why not SourceTree ?

Reply:
Because it says "apps we love" 🤣 you should really try Fork instead, trust me you'll love it 👍

Reply:
ok then create the section `Personal Preference` here 😛

Reply:
[GitUp](https://gitup.co) or GTFO 😂

Reply:
Source Tree should be included in this list. I like it, and so do other members.

Reply:
Ok, ok, just because it's a good wether today!